fastlane/swift/ArgumentProcessor.swift

```diff
@@ -64,6 +65,15 @@ struct ArgumentProcessor {
         } else {
             self.commandTimeout = SocketClient.defaultCommandTimeoutSeconds
         }
+        
+        var laneParameters = [String: String]()
```

Comment:
Let's extract this out to a new function. There is enough going on that having a new function would make it clearer.